generators/chipyard/src/main/scala/config/SodorConfigs.scala

```diff
@@ -0,0 +1,24 @@
+package chipyard
+
+import chisel3._
+
+import freechips.rocketchip.config.{Config}
+
+class SodorConfig extends Config(
+  new chipyard.iobinders.WithUARTAdapter ++
```

Comment:
Can this extend AbstractConfig?

Reply:
Seems like `AbstractConfig` contains configuration for external memory and L2 cache which there is no config fragment that can clear those, so I cannot use that class.

Reply:
`freechips.rocketchip.subsystem.WithNoExtMem` removes external memory.
`freechips.rocketchip.subsystem.WithNoBanks(0)` removes L2.

Reply:
Fixed. I use `freechips.rocketchip.subsystem.WithNMemoryChannel(0)` to remove the memory.